pkg/doc/verifiable/linked_data_proof.go

```diff
@@ -52,25 +51,20 @@ type keyResolverAdapter struct {
 	pubKeyFetcher PublicKeyFetcher
 }
 
-func (k *keyResolverAdapter) Resolve(id string) ([]byte, error) {
+func (k *keyResolverAdapter) Resolve(id string) (*verifier.PublicKey, error) {
```

Comment:
same here... add a TODO comment that says: replace `*verifier.PublickKey` with Tink's `*keyset.Handle`